operators/src/processing/expression.rs

```diff
@@ -241,12 +241,13 @@ where
         &'b self,
         query: QueryRectangle,
         ctx: &'b dyn QueryContext,
-    ) -> BoxStream<'b, Result<RasterTile2D<TO>>> {
+    ) -> Result<BoxStream<'b, Result<RasterTile2D<TO>>>> {
```

Comment:
wouldn't it makes ense to fail here now if e.g. the CL program could not be run or parameters could not be set? Instead of panic.

Reply:
Also here, that is inside the stream and has nothing to do with `query`.